inlong-sdk/sort-sdk/src/main/java/org/apache/inlong/sdk/sort/impl/InLongTopicManagerImpl.java

```diff
@@ -152,13 +176,15 @@ public boolean clean() {
             }
 
             closeFetcher();
-            result = true;
-            logger.info("close finished {} {}", sortTaskId, result);
+            closePulsarClient();
+            closeTubeSessionFactory();
```

Comment:
Is it better to move cache difference to InLongTopicFetcher instance?

Reply:
The closing logic here belongs to the manager, and the specific closing implementation is in the fetcher